src/PhpWord/Writer/HTML/Element/Title.php

```diff
@@ -46,8 +46,14 @@ public function write()
             $writer = new Container($this->parentWriter, $text);
             $text = $writer->write();
         }
+        $css = '';
+        $style = \PhpOffice\PhpWord\Style::getStyle('Heading_' . $this->element->getDepth());
```

Comment:
Could you define class in use function in the header of the file ?

Reply:
Done, and eliminated 2 other fully qualified names as well.